python/ml4ir/applications/ranking/tests/test_lr_schedule.py

```diff
@@ -0,0 +1,217 @@
+import unittest
+import warnings
+import os
+from ml4ir.base.io import logging_utils
+from ml4ir.base.io.local_io import LocalIO
+from ml4ir.base.features.preprocessing import *
+from ml4ir.base.model.optimizers import cyclic_learning_rate
+from ml4ir.base.config.keys import DataFormatKey, TFRecordTypeKey
+from ml4ir.base.data.relevance_dataset import RelevanceDataset
+from ml4ir.base.model.relevance_model import RelevanceModel
+from ml4ir.base.model.losses.loss_base import RelevanceLossBase
+from ml4ir.base.model.scoring.scoring_model import ScorerBase, RelevanceScorer
+from ml4ir.base.model.scoring.interaction_model import InteractionModel, UnivariateInteractionModel
+from ml4ir.base.model.optimizers.optimizer import get_optimizer
+from ml4ir.applications.ranking.model.ranking_model import RankingModel
+from ml4ir.applications.ranking.config.keys import LossKey
+from ml4ir.applications.ranking.config.keys import ScoringTypeKey
+from ml4ir.applications.ranking.model.losses import loss_factory
+import yaml
+from ml4ir.base.features.feature_config import ExampleFeatureConfig, SequenceExampleFeatureConfig
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+from tensorflow.keras.optimizers.schedules import ExponentialDecay
+
+warnings.filterwarnings("ignore")
+
+INPUT_DIR = "ml4ir/applications/ranking/tests/data/"
+MODEL_CONFIG = "ml4ir/applications/ranking/tests/data/config/model_config_cyclic_lr.yaml"
+KEEP_ADDITIONAL_INFO = 0
+NON_ZERO_FEATURES_ONLY = 0
+
+class lr_callback(keras.callbacks.Callback):
```

Comment:
@mohazahran PEP8 recommends we use CapWords convention for class names -> https://www.python.org/dev/peps/pep-0008/#class-names